pages/cart-page/cart-page.css

```diff
@@ -0,0 +1,72 @@
+@import url('/main.css');
```

Comment:
You can maintain generic file and you can add all css files there instead of importing in each and every file

Reply:
yes, but I have kept all screens independent here.